src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3363,8 +3345,7 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
         gallery_name=gallery_name,
         gallery_image_name=gallery_image_name,
         gallery_image_version_name=gallery_image_version,
-        gallery_image_version=image_version,
-        headers={'x-ms-authorization-auxiliary': external_bearer_token}
```

Comment:
You deleted lots of code in vm/custom.py and didn't add anything. How are external tokens handled now? Is another PR required to fix it?

Reply:
> You deleted lots of code in vm/custom.py and didn't add anything.

The logic is not deleted, but moved to `_get_mgmt_service_client`. 

> How are external tokens handled now? 

External tokens are handled automatically when `aux_subscriptions` is provided to `_get_mgmt_service_client`.

```py
client = _compute_client_factory(cmd.cli_ctx, aux_subscriptions=aux_subscriptions)
```

Is another PR required to fix it?

> It already works. No other PR needed.